plumbing/format/packfile/parser.go

```diff
@@ -256,8 +279,14 @@ func (p *Parser) indexObjects() error {
 			copy(p.deltas[oh.Offset], data)
 		}
 
+		if delta {
+			p.deltasTotal++
+		}
```

Comment:
🍜 We count the total of deltas in the packfile here so that we can track progress while resolving them later.